src/eko/anomalous_dimensions/as1aem1.py

```diff
@@ -15,7 +15,7 @@
 
 
 @nb.njit("u1(u1)", cache=True)
-def number_of_up_flavors(nf):
+def uplike_flavors(nf):
```

Comment:
I think you should lift that function ... e.g. to `__init__` as it has a broader scope

Reply:
I have moved it in constants.py since I think that can be used also outside anomalous_dimension.
What do you think?